app/models/webhooks/slack/new_vote.rb

```diff
@@ -0,0 +1,49 @@
+class Webhooks::Slack::NewVote < Webhooks::Slack::Base
+
+  def text
+    I18n.t :"webhooks.slack.new_vote", author: author.name, position: vote_position, name: discussion_link, proposal: proposal_link(eventable)
+  end
+
+  def attachment_fallback
+    "*#{eventable.position}*\n#{eventable.statement}\n"
+  end
+
+  def attachment_title
+  end
+
+  def attachment_text
+    "#{eventable.statement}\n"
+  end
+
+  def attachment_fields
+  end
+
+  def attachment_color
+    case eventable.position
+    when "yes"
+      "#94D587"
+    when "no"
+      "#D1908F"
+    when "abstain"
+      "#EEBC57"
+    else
+      "#D80D00"
+    end
+  end
+
+  private
+
+  def vote_position
+    case eventable.position
+    when "yes"
```

Comment:
I like one-liney case syntax :palm_tree: 

```
when "no" then "disagreed on"
```

I want to flag that there's a translation component here (don't worry about it in this PR), but that we do want to, at some point, allow for translations of these things, likely by allowing Webhooks to have a locale associated with them.